app/api/v2/views/user.py

```diff
@@ -17,21 +19,21 @@ def wrapper(*args, **kwargs):
         return fn(*args, **kwargs)
     return wrapper
 
-def candidate_required(fn):
-    @wraps(fn)
-    def wrapper(*args, **kwargs):
-        jwt_holder()
-        if get_jwt_claims()['isCandidate'] != True:
-            return make_response(jsonify({"message": "Candidate rights required!"}), 201)
-            pass
-        return fn(*args, **kwargs)
-    return wrapper
+# def candidate_required(fn):
+#     @wraps(fn)
+#     def wrapper(*args, **kwargs):
+#         jwt_holder()
+#         if get_jwt_claims()['isCandidate'] != True:
+#             return make_response(jsonify({"message": "Candidate rights required!"}), 201)
+#             pass
+#         return fn(*args, **kwargs)
+#     return wrapper
+
```

Comment:
Remove the commented out code if you don't need it.

Reply:
I will, thanks